configs/knative_yamls/serving-default-domain.yaml

```diff
@@ -33,8 +33,8 @@ spec:
         - name: default-domain
           # This is the Go import path for the binary that is containerized
           # and substituted here.
-          image: docker.io/vhiveease/default-domain-dd6f55160290a90018ba318b624be12e:998c8975f56b3b45059fd709d3776062b51cb1aa84921d4d7a85c0315d9cd0f0
-          args: ["-magic-dns=nip.io"]
+          image: docker.io/vhiveease/default-domain-dd6f55160290a90018ba318b624be12e@sha256:b9afac8d6bba3a4d055b7b636bcc6d6e61feb1e3936728cea09dabaea50648ca
+          args: ["-magic-dns=sslip.io"]
```

Comment:
need to replace `xip.io` in the CRI tests (grep in the `cri/` folder) with sslip.io